notebook/tests/launchnotebook.py

```diff
@@ -39,11 +39,12 @@ class NotebookTestBase(TestCase):
 
     port = 12341
     config = None
+    url_prefix = '/a%40b/'
```

Comment:
Ah - `/a@b/` is just an arbitrary unique string you picked to prefix testing URLs?